src/views/viewer/ChannelView/ChannelAbout.tsx

```diff
@@ -22,8 +21,8 @@ export const ChannelAbout = () => {
   const { channel } = useChannel(id)
   const { videoCount } = useChannelVideoCount(id)
   return (
-    <Container>
-      <div>
+    <StyledLayoutGrid>
```

Comment:
Spacing on mobile in information tab is wrong (between tabs and content). This was here before but let's fix it now
<img width="408" alt="Screenshot 2021-08-13 at 11 33 48" src="https://user-images.githubusercontent.com/12646744/129337281-f91eb4c3-fee8-4252-bc50-4a47cdf76caf.png">